view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -137,7 +255,7 @@ define(
              * @return {boolean}
              */
             orderValueTooLow: function () {
-                return window.checkoutConfig.totalsData.base_grand_total < 3000;
+                return this.getTotal() < 3000;
```

Comment:
@jacstn regarding to the Magento's implementation and the way they use `base_total`, `total`, `base_grand_total`, `grand_total`. Having a method that change the definition of `base_grand_total` to `getTotal` may lead to a misunderstanding?

Reply:
i think grand total is better than base grand total according to this article:
https://stackoverflow.com/questions/17999330/what-is-the-difference-among-base-grand-total-grand-total-sub-total-in-magent

because grand total is always displaying in shop main currency, where base_grand_total keeps value in current chosen currency (if user defined one)

Reply:
i return there `grand_total`, changed from `base_grand_total`

Reply:
But then this amount won't be accurated right?
I.e. A store base currency is `THB` and the display currency I chose is `USD`.
When I pick an item that its amount is 100 USD at `grand_total`, (meaning, about 3,1xx THB at `base_grand_total`) but this store will show me that my purchase amount is too low and I have to pick more item to make its amount exceed 3,000 USD (at `grad_total`) which way far more exceed than 3,000 THB at `base_grand_total`.

Reply:
For the question
> @guzzilar did you test it previous grand_total comment? we force THB, no THB installment will be not selectable. But might be some my mistake here..

Let me move to this thread.

My fault, yup, it's protected by `isActive()` function already.
However, one little note.
<img width="1552" alt="Screen Shot 2562-05-24 at 22 00 41" src="https://user-images.githubusercontent.com/2154669/58341896-83ddbb00-7e79-11e9-8a83-0a8e4f0d1ee2.png">

It would be nice if we can hide that `order value too low` message if the method isn't active (actually, how about `order amount` instead of `order value`).

Maybe `visible: (isActive() && orderValueTooLow())` could help?
Anyway, feel free to click resolve this conversation if you don't have anything to add. (about the warning message, I think we can do it later as it is out of the scope of this PR)

Reply:
The formatting of THB amounts doesn't seem to be following Magento settings

Reply:
I changed formatting @jonrandy in next PR #202
@guzzilar also your condition is added there with showing that message

Reply:
👍 Awesome! Resolved :D